.zombienet-tests/0002-getHeaders.js

```diff
@@ -0,0 +1,19 @@
+const { connect } = require("./utils")
+
+async function run(nodeName, networkInfo) {
+  const api = await connect(nodeName, networkInfo)
+  // add 20s sleep to give time to sync
+  await new Promise((resolve) => setTimeout(resolve, 10000))
```

Comment:
Are we sure there's no better way? This is really bad

Reply:
And if there's no better way, please add a comment explaining precisely why this is necessary